app/models/ticket_purchase.rb

```diff
@@ -58,4 +60,11 @@ def self.update_quantity(conference, quantity, ticket, user)
     purchase.quantity = quantity if quantity > 0
     purchase
   end
+
+  def pay(payment)
+    update_attributes(paid: true, payment_id: payment)
```

Comment:
No I think we have to specify the exact attribute name in update_attributes. I tried this and it gives an error unknown attribute payment.